pages/items/[id]/index.js

```diff
@@ -11,11 +11,15 @@ export default function DetailsPage() {
 
   async function handleStatus() {
     const newItem = { ...item, inDiscuss: !item.inDiscuss };
```

Comment:
I would rename this to "updatedItem" to clarify that you change an already existing item.